### ISSUE TITLE
Command key in API docs

### ISSUE THREAD

**Issue by AsLogd** [microsoft/monaco-editor#1342]
The API docs seems to be capturing the Command Key (start key/Windows key). Whenever I press it, the search bar is focused. This is unfortunate as I'm sure a lot of users use this key with i3 desktops to navigate. Every time I have to change context between the code and the API docs I find myself at the top of the page which is annoying.
Also, wouldn't Windows users also use this key to start programs for instance? What's the logic behind using this key for this purpose?

tl;dr: 
In API docs:
Action: Command key
Behaviour: Search bar is focused
Expected: Nothing
Proposal: Change binding or allow users to disable ir

**Comment by alexandrudima** [comment]
@AsLogd I cannot reproduce this. I have tried at https://microsoft.github.io/monaco-editor/api/index.html on a mac and by pressing the command key. Nothing happens.

**Comment by AsLogd** [comment]
@alexandrudima Hmm I see. Thanks for the input. I don't have a Mac but I definitely have a Windows. I'll try there later too and see if it's something Linux-specific

**Comment by alexdima** [comment]
@AsLogd It looks to me that you are hitting some code in a Chrome extension that you have installed? The string `"hotkey-pressed"` does not appear anywhere in the JS code we load on our website.
